### ISSUE TITLE
Add an area on the website for businesses that support/use the ETC blockchain

### ISSUE THREAD

**Issue by justjinn** [ethereumclassic/ethereumclassic.github.io#768]
Was looking over the site and realized businesses as, NOWPayments.io, some travel agency (don't remember the business name), and others are not on the site.

**Comment by brolal** [comment]
yeah, good point. would be nice to have a dedicated page for businesses and merchants

**Comment by justjinn** [comment]
The travel company is Travala.com

**Comment by brolal** [comment]
I think Classic Kevin knows most merchants on ETC

**Comment by w1g0** [comment]
@justjinn, businesses that accept payments in $ETC would make a useful section on the website.
